src/components/exercise-scenarios/ExerciseScenarios.js

```diff
@@ -1,96 +1,125 @@
-    week: 2,
-    type: 'Øvelse',
-    dfu: 4,
-    scenario: 'Fartøy på kollisjonskurs'
-  },
-  {
-    id: 1,
-    week: 3,
-    type: 'Table top',
-    dfu: 15,
-    scenario: 'Ekstremvær over design-/operasjonskriterier'
-  },
-  {
-    id: 2,
-    week: 8,
-    type: 'Øvelse',
-    dfu: 11,
-    scenario: 'Alvorlig syk/skadet person'
-  },
-  {
-    id: 3,
-    week: 9,
-    type: 'Table top',
-    dfu: 7,
-    scenario: 'Helikopterulykke i sjø'
-  },
-  {
-    id: 4,
-    week: 14,
-    type: 'Øvelse',
-    dfu: 10,
-    scenario: 'Mann over bord'
-  },
-  {
-    id: 5,
-    week: 15,
-    type: 'Table top',
-    dfu: '12',
-    scenario: 'Sikringstrussel/terror'
-  },
-  {
-    id: 6,
-    week: 20,
-    type: 'Øvelse',
-    dfu: '9',
-    scenario: 'Brann i boligkvarter'
-  },
-  {
-    id: 7,
-    week: 21,
-    type: 'Table top',
-    dfu: '14',
-    scenario: 'Strukturhendelse'
-  }
+const rows = [
+  { id: "week", numeric: true, label: "Uke" },
+  { id: "type", numeric: false, label: "Type" },
+  { id: "dfu", numeric: true, label: "DFU" },
+  { id: "scenario", numeric: false, label: "Scenario" },
 ];
 
-export default () => (
-  <AppPage title="Øvelsesscenarioer" back="..">
-    <Paper style={{margin: '1em'}}>
-      <Table>
-        <TableHead>
-          <TableRow>
-            <TableCell>Uke</TableCell>
-            <TableCell>Type</TableCell>
-            <TableCell>DFU</TableCell>
-            <TableCell>Scenario</TableCell>
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {scenarios.map(row => {
+class ExerciseScenariosTableHead extends React.Component {
+  createSortHandler = property => event => {
+    this.props.onRequestSort(event, property);
+  };
+
+  render() {
+    const { order, orderBy } = this.props;
+
+    return (
+      <TableHead>
+        <TableRow>
+          {rows.map(row => {
             return (
-              <TableRow key={row.id}>
-                <TableCell>{row.week}</TableCell>
-                <TableCell>{row.type}</TableCell>
-                <TableCell>{row.dfu}</TableCell>
-                <TableCell>{row.scenario}</TableCell>
-              </TableRow>
+              <TableCell
+                key={row.id}
+                numeric={row.numeric}
+                sortDirection={orderBy === row.id ? order : false}
+              >
+                <Tooltip
+                  title="Sortér"
+                  placement={row.numeric ? "bottom-end" : "bottom-start"}
+                  enterDelay={300}
+                >
+                  <TableSortLabel
+                    active={orderBy === row.id}
+                    direction={order}
+                    onClick={this.createSortHandler(row.id)}
+                  >
+                    {row.label}
+                  </TableSortLabel>
+                </Tooltip>
+              </TableCell>
             );
-          })}
-        </TableBody>
-      </Table>
-    </Paper>
-  </AppPage>
-);
+          }, this)}
```

Comment:
When using arrow function supplying a `this` context as done with `.map(..., this)` here is unnecessary. The arrow function captures the `this` context, so `, this` can be safely removed.